api/v1alpha1/managedcluster_types.go

```diff
@@ -114,7 +114,7 @@ type ManagedClusterStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +kubebuilder:resource:shortName=hmc-deploy;deploy
+// +kubebuilder:resource:shortName=mcluster;mcl
```

Comment:
i have already done this in the #422 but the names are different (`hmcmc;mc`), if the latter are ok maybe we could wait for the merge, otherwise LGTM

Reply:
@zerospiel Tbh I don't have any preferences on the naming.
But I guess I will merge this to provide correct changes to the docs which I'm doing now.
Thanks!